aibolit/metrics/number_variables/numVariables.py

```diff
@@ -0,0 +1,26 @@
+from aibolit.utils.ast_builder import build_ast
+from javalang.tree import VariableDeclarator, ConstructorDeclaration
+
+
+class NumVars:
+    '''
+    This class count number of methods and variables in class
```

Comment:
wrong comment: we do not calculate the number of methods

Reply:
yeap, sorry, forgot to rewrite it